src/main/java/frc/robot/subsystems/lifter/ArmSubsystem.java

```diff
@@ -40,10 +43,12 @@ public ArmSubsystem() {
         m_controller.enableContinuousInput(0, Math.PI*2);
     }
 
-    // commands
     public Command keepInPlaceCommand() {
-        return new RunCommand(
-            () -> m_IO.setVoltage(0), 
+        return new FunctionalCommand(
+            () -> {}, 
+            () -> m_IO.setVoltage(0),
+            (interrupted) -> {},
+            () -> !isAtPosition(m_goal.position),
```

Comment:
This ends when you are not at the position but a new command doesn't run AFAICT.

I would right a new keepInPlacePidCommand, which just runs PID on the current goal.